ci-operator/jobs/openshift/cluster-etcd-operator/openshift-cluster-etcd-operator-master-presubmits.yaml

```diff
@@ -1067,6 +1067,136 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-metal-ipi,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - ^master$
+    - ^master-
+    cluster: build01
+    context: ci/prow/e2e-metal-ipi-ovn-ipv6
+    decorate: true
+    labels:
+      ci-operator.openshift.io/prowgen-controlled: "true"
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-openshift-cluster-etcd-operator-master-e2e-metal-ipi-ovn-ipv6
```

Comment:
Could you make this optional please?

Reply:
Thanks for review @lilic. You are right, it is better to set them as optional. I'll change in this way.